questions.md

```diff
@@ -0,0 +1,4 @@
+# Questions about Pull Requests
+
+* Should the student want to add more files, would clone be the best option?
```

Comment:
👋 Hi @waldonhendricks ! If a student would like to add more files to a repository, they are welcome to do so by either merging those changes to master locally and pushing those changes to the remote branch (not ideal method) or creating a Pull Request with file changes, so it can be reviewed and then merged when it is ready. In this case the student would be a collaborator on the repository.